### DISCUSSION TITLE
怎么导入电脑的rime?

### DISCUSSION THREAD

**Discussion by jerryc127** [imfuxiao/Hamster#186] [Q&A]
我想用雾凇拼音
然后用输入方案上传整个rime文件夹

https://github.com/iDvel/rime-ice

但是部署后，效果不一样，不能正常打字。

还有 怎么切换 繁体输出？

**Comment by imfuxiao** [comment]
抱歉，1.6.4版本有bug，导致键盘启动时crash。您看是否可以先用TF版本临时过渡下，1.6.5会修复此问题。

[雾淞输入法导入教程](https://github.com/imfuxiao/Hamster/wiki/%E5%A6%82%E4%BD%95%E5%AF%BC%E5%85%A5%22%E9%9B%BE%E6%B7%9E%E6%8B%BC%E9%9F%B3%E8%BE%93%E5%85%A5%E6%96%B9%E6%A1%88%22)

[TF内测链接](https://testflight.apple.com/join/6AkRfQYZ)
